packages/schematics/src/collection/node-application/index.ts

```diff
@@ -0,0 +1,270 @@
+    return {
+      ...json,
+      projects: {
+        ...json.projects,
+        [options.name]: { tags: options.parsedTags }
+      }
+    };
+  });
+}
+
+function getBuildConfig(project: any, options: NormalizedSchema) {
+  return {
+    builder: '@nrwl/builders:node-build',
+    options: {
+      outputPath: join(normalize('dist'), options.appProjectRoot),
+      main: join(project.sourceRoot, 'main.ts'),
+      tsConfig: join(options.appProjectRoot, 'tsconfig.app.json')
+    },
+    configurations: {
+      production: {
+        optimization: true,
+        extractLicenses: true,
+        fileReplacements: [
+          {
+            replace: join(project.sourceRoot, 'environments/environment.ts'),
+            with: join(project.sourceRoot, 'environments/environment.prod.ts')
+          }
+        ]
+      }
+    }
+  };
+}
+
+function getServeConfig(options: NormalizedSchema) {
+  return {
+    builder: '@nrwl/builders:node-execute',
+    options: {
+      buildTarget: `${options.name}:build`
+    }
+  };
+}
+
+function updateAngularJson(options: NormalizedSchema): Rule {
+  return updateJsonInTree('angular.json', angularJson => {
+    const project = angularJson.projects[options.name];
+    const fixedProject = replaceAppNameWithPath(
+      project,
+      options.name,
+      options.appProjectRoot
+    );
+    delete fixedProject.architect.test;
+
+    fixedProject.architect.build = getBuildConfig(fixedProject, options);
+    fixedProject.architect.serve = getServeConfig(options);
+
+    fixedProject.architect.lint.options.tsConfig = fixedProject.architect.lint.options.tsConfig.filter(
+      path =>
+        path !== join(normalize(options.appProjectRoot), 'tsconfig.spec.json')
+    );
+    angularJson.projects[options.name] = fixedProject;
+
+    delete angularJson.projects[options.e2eProjectName];
+    return angularJson;
+  });
+}
+
+function removeE2e(options: NormalizedSchema): Rule {
+  return (host: Tree) => {
+    [
+      'protractor.conf.js',
+      'tsconfig.e2e.json',
+      'src/app.e2e-spec.ts',
+      'src/app.po.ts'
+    ].forEach(path => {
+      host.delete(join(options.e2eProjectRoot, path));
+    });
+    return host;
+  };
+}
+
+function removeAppFiles(options: NormalizedSchema): Rule {
+  return (host: Tree) => {
+    [
+      'src/favicon.ico',
+      'src/index.html',
+      'src/polyfills.ts',
+      'src/styles.css',
+      'browserslist',
+      'src/app/app.module.ts',
+      'src/app/app.component.ts'
+    ].forEach(path => {
+      host.delete(join(options.appProjectRoot, path));
+    });
+    return host;
+  };
+}
+
+function removeKarma(options: NormalizedSchema): Rule {
+  return (host: Tree) => {
+    host.delete(join(options.appProjectRoot, 'src/test.ts'));
+    host.delete(join(options.appProjectRoot, 'tsconfig.spec.json'));
+    host.delete(join(options.appProjectRoot, 'karma.conf.js'));
+    return host;
+  };
+}
+
+export default function(schema: Schema): Rule {
```

Comment:
Since aren't benefiting from the angular application that much, I think it's better to generate all the files ourselves instead of invoking the application schematic and then patching and removing so man files.